### ISSUE TITLE
[A11Y] "Skip access checks" checkbox not reachable by keyboard

### ISSUE THREAD

**Issue by foxbunny** [indico/indico#5952]
**Describe the bug**

The "Skip access checks" checkbox cannot be reached by tab navigation.

**To Reproduce**
1. Go to main page as an admin user
2. Enter some text in the search box
3. Use the Tab key to attempt to focus the checkbox
4. Observe that checkbox is skipped in focus order

**Expected behavior**

Checkbox should receive focus right after the search box

**Additional context**

- https://www.w3.org/WAI/WCAG21/Understanding/keyboard

**Comment by foxbunny** [comment]
This issue is caused by a bug in Semantic UI. [Reported upstream](https://github.com/Semantic-Org/Semantic-UI-React/issues/4440).
